.github/workflows/pipeline_test.yaml

```diff
@@ -90,11 +90,14 @@ jobs:
 
           while True:
             status = client.get_run(run_id=run_id).state
-            if status not in ["SUCCEEDED", "FAILED", "ERROR"]:
+            if status in ["PENDING", "RUNNING"]:
```

Comment:
@kromanow94 a bit late to request this change, but could we not have the python code directly on the GH action but convert to a `.py` file, and use that from the action?

This will allow users to also run the tests locally if they want

Reply:
I think it makes a lot of sense. Do you mind if I do it next week?

Reply:
just want to say I also vote for moving to a `.py` file.

Reply:
I think this is a part of a wider topic to reorganize tests. I created an issue for that:

https://github.com/kubeflow/manifests/issues/2797